pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/migration/CohortTagMigrator.java

```diff
@@ -16,6 +20,25 @@ public CohortTagMigrator(String index, String realm, CohortTagDao cohortTagDao)
 
     @Override
     protected Map<String, Object> getDataByRealm() {
-        return (Map) cohortTagDao.getCohortTagsByInstanceName(realm);
+        Map<String, List<CohortTag>> cohortTags = cohortTagDao.getCohortTagsByInstanceName(realm);
+        updateCohortTagsIfRequired(cohortTags);
```

Comment:
this would not be enough?
``` 
AdditionalCohortTagsRetriever.fromRealm(realm)
                .ifPresent(retriever -> cohortTags.putAll(retriever.retrieve()));
```